content/tbb/tbb-14/contents.lr

```diff
@@ -6,4 +6,17 @@ seo_slug: installing-add-on-extensions-tor-browser
 ---
 description:
 It's strongly discouraged to install new add-ons in Tor Browser, because they can compromise your privacy and security.
+
+How?
+Basically, each browser's settings and features create what is called a "browser fingerprint".
+Most browsers inadvertently create a unique fingerprint for each user which can be tracked across the internet.
+Tor Browser is specifically engineered to have a nearly identifcal (we're not perfect!) fingerprint across it's users.
+This means each Tor Browser user looks like every other Tor Browser user, making it difficult to track any individual user.
+
+Now, how does this relate to installing new add-ons in Tor Browser?
```

Comment:
Let's remove the questions "How?" and "Now, how does this relate to installing new add-ons in Tor Browser" to make reading more fluid.